tests/component/test_interpreter.py

```diff
@@ -40,6 +39,8 @@ def test___grpc_channel_with_errors___get_error_string___returns_failed_to_retri
 def test___known_error_codes___get_error_string_with_error_code___returns_matching_error_message(
     grpc_init_kwargs, error_code: int
 ):
+    from nidaqmx._grpc_interpreter import GrpcStubInterpreter
+
     interpreter = GrpcStubInterpreter(**grpc_init_kwargs)
```

Comment:
FYI, I'm not a fan of how `black` adds a newline here, because it splits the test function into 4 sections (import, arrange, act, assert). Perhaps moving the import inside the try block is slightly better. However, it's not a dealbreaker.

Reply:
Moved to `try` block.